File: samples/conference/registration-bc/src/main/java/com/microsoft/conference/registration/readmodel/QueryServices/OrderSeatAssignment.cs
using System;
namespace Registration.ReadModel
{
    public class OrderSeatAssignment
    {
        public Guid AssignmentsId { get; set; }
        public int Position { get; set; }
        public string SeatTypeName { get; set; }
        public string AttendeeEmail { get; set; }
        public string AttendeeFirstName { get; set; }
        public string AttendeeLastName { get; set; }
    }
}

File: samples/conference/registration-bc/src/main/java/com/microsoft/conference/registration/readmodel/service/OrderSeatAssignment.java
package com.microsoft.conference.registration.readmodel.service;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class OrderSeatAssignment {
    private String assignmentsId;
    private int position;
    private String seatTypeName;
    private String attendeeEmail;
    private String attendeeFirstName;
    private String attendeeLastName;
}
